### ISSUE TITLE
Issue with `databricks_sql_visualization` resource

### ISSUE THREAD

**Issue by paresh2710** [databricks/terraform-provider-databricks#1623]
### Configuration
```hcl
# Copy-paste your Terraform configuration here

resource "databricks_sql_visualization" "this" {
  for_each = {
    for k in var.visualization_list : k.name => k
  }
  query_id    = databricks_sql_query.this[0].id
  name        = each.key
  type        = each.value.type
  description = each.value.description

  options = jsonencode(each.value.options)
}

visualization_list = [

      {

        "type": "CHART",

        "name": "Quarantine History Count",

        "description": "",

        "options": {

            "version": 2,

            "globalSeriesType": "line",

            "sortX": true,

            "sortY": true,

            "legend": {

                "traceorder": "normal"

            },

            "xAxis": {

                "type": "-",

                "labels": {

                    "enabled": true

                }

            },

            "yAxis": [

                {

                    "type": "-"

                },

                {

                    "type": "-",

                    "opposite": true

                }

            ],

            "alignYAxesAtZero": false,

            "error_y": {

                "type": "data",

                "visible": true

            },

            "series": {

                "stacking": null,

                "error_y": {

                    "type": "data",

                    "visible": true

                }

            },

            "seriesOptions": {

                "column_97b3c9e92": {

                    "yAxis": 0,

                    "type": "line"

                }

            },

            "valuesOptions": {},

            "direction": {

                "type": "counterclockwise"

            },

            "sizemode": "diameter",

            "coefficient": 1,

            "numberFormat": "0,0[.]00000",

            "percentFormat": "0[.]00%",

            "textFormat": "",

            "missingValuesAsZero": true,

            "useAggregationsUi": true,

            "swappedAxes": false,

            "dateTimeFormat": "YYYY-MM-DD HH:mm",

            "showDataLabels": true,

            "columnConfigurationMap": {

                "x": {

                    "column": "EXTRACT_DTE",

                    "id": "column_97b3c9e91"

                },

                "y": [

                    {

                        "id": "column_97b3c9e92",

                        "column": "QUARANTINE_RECORD_COUNT",

                        "transform": "SUM"

                    }

                ],

                "series": {

                    "column": "TABLE_NAME",

                    "id": "column_48cd299f1"

                }

            },

            "isAggregationOn": true,

            "condensed": true,

            "withRowNumber": true

        },

        "query_plan": {

            "selects": [

                {

                    "column": "EXTRACT_DTE"

                },

                {

                    "function": "SUM",

                    "args": [

                        {

                            "column": "QUARANTINE_RECORD_COUNT"

                        }

                    ],

                    "alias": "column_97b3c9e92"

                },

                {

                    "column": "TABLE_NAME"

                }

            ],

            "groups": [

                {

                    "column": "EXTRACT_DTE"

                },

                {

                    "column": "TABLE_NAME"

                }

            ]

        }

      }

]

### Expected Behavior
<!-- What should have happened? -->
query plan applied on DB dashboard visualisation and graphs(bar charts) are rendered as per query plan(x axis.., y axis...)

### Actual Behavior
<!-- What actually happened? -->
DB Dashboard visualisation rendered through terraform has no query plan hence no bar-chart visible just a st. line(x axis)
(query: above script, is the query_plan placed correctly??? or it has to be added differently???)

### Steps to Reproduce
<!-- Please list the steps required to reproduce the issue, for example:
1. `terraform apply`-->
a. Manually create visualisation(API) with query plan, render the dashboard
b. Now remove query plan from the above step and verify the dashboard with visualisation
c. Next, render the dashboard visualisation with terraform (#b should be same as #c)

Create the dashboard using terraform and apply aggregation query associated with query plan

### Terraform and provider versions
<!-- Please paste the output of `terraform version`. If version of `databricks` provider is not the latest (https://github.com/databricks/terraform-provider-databricks/releases), please make sure to use the latest one. -->
terraform 1.3.0
databricks terraform 1.2.1

### Debug Output
<!-- Please add turn on logging, e.g. `TF_LOG=DEBUG terraform apply -no-color` and run command again, paste it to gist & provide the link to gist. If you're still willing to paste in log output, make sure you provide only relevant log lines with requests. It would make it more readable, if you pipe the log through `| grep databricks | sed -E 's/^.* plugin[^:]+: (.*)$/\1/'`, e.g.: `TF_LOG=DEBUG terraform apply -no-color 2>&1 | grep databricks | sed -E 's/^.* plugin[^:]+: (.*)$/\1/' 2>&1 |tee tf-debug.log`. If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`. -->

### Important Factoids
<!-- Are there anything atypical about your accounts that we should know? -->

**Comment by nfx** [comment]
@pietern can you look at this one?

**Comment by pietern** [comment]
The `query_plan` attribute isn't available in the `databricks_visualization` resource and needs to be added.

**Comment by nfx** [comment]
Following up - is this issue still relevant?

**Comment by alexott** [comment]
it was fixed in 1.6.5 via #1752
